src/components/Dashboard/Searchbar.tsx

```diff
@@ -4,32 +4,40 @@ import { splitNSearch } from '@/utils/splitNSearch';
 
 function Searchbar({ label }: searchProps) {
     const [query, setQuery] = useState('');
+    let btnStyle = styles.btnActive;
+    let btnDisabled = false;
 
     const handleKeyPress = (event: KeyboardEvent) => {
         if (event.key === 'Enter') {
             splitNSearch(query);
         }
     };
 
+    if (query === '') {
+        btnStyle = styles.btnDisabled;
+        btnDisabled = true;
+    }
+
     return (
         <section className={styles.container}>
-            <label htmlFor="search" className={styles.searchLabel}>
-                {label}
-            </label>
             <input
+                data-testid="searchbar_input"
                 type="text"
                 id="search"
                 value={query}
                 onChange={(e) => setQuery(e.target.value)}
                 onKeyDown={(e) => handleKeyPress(e)}
                 className={styles.searchBar}
+                placeholder={label + ':'}
             />
             <button
+                data-testid="search_btn"
                 type="submit"
                 onClick={() => {
                     splitNSearch(query);
```

Comment:
NIT: I think we can name this function better. Like `splitSearchByComma` or `splitSearch()` and pass the delimeter `,` as an argument

Reply:
THe method defined is not a part of current PR, I can surely take it up over the next/refactoring PR specific to it